smartsim/wlm/slurm.py

```diff
@@ -253,6 +253,8 @@ def _get_alloc_cmd(nodes: int,
                     salloc_args += [prefix + opt, str(val)]
                 else:
                     salloc_args += ["=".join((prefix + opt, str(val)))]
+        else:
```

Comment:
Line 239 of this file has a ``TODO``.  Can we capture that in a ticket and remove that ``TODO`` comment?